contracts/NestedFactory.sol

```diff
@@ -558,4 +561,17 @@ contract NestedFactory is INestedFactory, ReentrancyGuard, Ownable, MixinOperato
         _transferFeeWithRoyalty(feeAmount, _token, _nftId);
         _token.safeTransfer(_dest, _amount - feeAmount);
     }
+
+    /// @dev Verify that msg.value is equal to the amount needed (in the orders)
+    /// @param _batchedOrders The batched input orders
+    function _checkMsgValue(BatchedInputOrders[] calldata _batchedOrders) private {
+        uint256 ethNeeded;
+        uint256 length = _batchedOrders.length;
```

Comment:
💅 the length caching is not required here

Reply:
Changed 👍